### PR TITLE
Updated Installable version of pygdal

### PR DESCRIPTION
**Pull request by jvanderleeuw**
In order to allow agpypeline to be installed without conflicting pygdal versions, the tagged version can be updated from 0.0.21 to 0.0.34. Additionally, in order to avoid discrepancies between pygdal and the default gdal version (which appears to be installed separately than the pip installations), different versions of agpypeline are currently being used by transformer-opendronemap's dockerfile and pylint_check.yaml file.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -36,7 +36,7 @@ RUN apt-get update && \
     python3 -m pip install --upgrade --no-cache-dir \
         numpy && \
     python3 -m pip install --upgrade --no-cache-dir \
-        pygdal==2.2.3.5 && \
+        pygdal==3.1.1.6 && \
```
Comment (Chris-Schnaufer) [inline Dockerfile:39]: I am not able to make this work when building a Docker image

File: setup.py [hunk 2]
```diff
@@ -25,6 +25,6 @@
     ],
     python_requires='>=3.7',
     install_requires=['setuptools', 'numpy', 'influxdb', 'laspy', 'requests>=2.21.0', 'python-dateutil', 'utm',
-                      'matplotlib', 'Pillow', 'scipy', 'piexif', 'cryptography', 'pyyaml', 'pygdal==2.2.3.5']
+                      'matplotlib', 'Pillow', 'scipy', 'piexif', 'cryptography', 'pyyaml', 'pygdal==2.2.2.*',]
```
Comment (Chris-Schnaufer) [inline setup.py:28]:
```suggestion
                      'matplotlib', 'Pillow', 'scipy', 'piexif', 'cryptography', 'pyyaml', 'pygdal>=2.2.2',]
```

**Comment by jvanderleeuw** [comment]
The default version after making the changes is now 0.0.36

File: .github/workflows/pypi_push_workflow.yaml [hunk 3]
```diff
@@ -2,27 +2,32 @@ on:
   push:
     branches:
       - master
-
+      - testing
```
Comment (Chris-Schnaufer) [inline .github/workflows/pypi_push_workflow.yaml:5]:
```suggestion
```

Comment (Chris-Schnaufer) [inline .github/workflows/pypi_push_workflow.yaml:5]:
```suggestion
```

**Pull request merged**